### PR TITLE
fix(slot exporter): improve validator online / offline detection

### PR DESCRIPTION
**Pull request by peterbitfly**

### CODE REVIEW

File: backend/pkg/exporter/modules/slot_exporter.go [hunk 1]
```diff
@@ -619,54 +668,6 @@ func ExportSlot(client rpc.Client, slot uint64, isHeadEpoch bool, tx *sqlx.Tx) e
...
-	}
-
-	// save the proposal to bigtable
-	err = db.BigtableClient.SaveProposal(block)
-	if err != nil {
-		return fmt.Errorf("error exporting proposal to bigtable for slot %v: %w", block.Slot, err)
-	}
-
-	// save the block data to the db
-	err = edb.SaveBlock(block, false, tx)
```
Comment (Eisei24) [inline backend/pkg/exporter/modules/slot_exporter.go:660]: **issue:** This part was also moved before the big if but we change the content of block within the if, e.g. in the currently adjusted `SaveValidators`.
AFAIK we only wanted to move the bigtable duties to the front.

If a `SaveBlock` at the start is fine please let me know.

**Review by Eisei24 — COMMENTED** [review]
Please check the changed `SaveBlock` handling.

File: backend/pkg/exporter/modules/slot_exporter.go (hunk 1 above)
Comment (peterbitfly) [reply backend/pkg/exporter/modules/slot_exporter.go:660]: i think its fine as we don't touch the mutated fields in the `SaveBlock` function. if you found a place where `SaveBlock` touches one of those fields please let me know.

**Review by Eisei24 — APPROVED** [review]
lgtm

**Pull request merged**